flexmeasures/data/services/scheduling.py

```diff
@@ -241,6 +246,99 @@ def create_scheduling_job(
     return job
 
 
+def cb_done_sequential_scheduling_job(jobs_ids: list[str]):
+    """
+    TODO: add logic
+    """
+
+    # jobs = [Job.fetch(job_id) for job_id in jobs_ids]
+    pass
+
+
+def create_sequential_scheduling_job(
+    asset_or_sensor: Asset | Sensor | None = None,
```

Comment:
This cannot be `None` anymore (we only had that in `create_scheduling_job` because of needing to deal with deprecating the `sensor` kwarg (I removed the `sensor` kwarg from this new method just now, as it's not needed).

I'm also doubting whether we need to support this also being a sensor, or whether it *must* always be an asset. Or is there a use case for sequential scheduling on a single sensor?